### ISSUE TITLE
Applying scaling transformations to point lights doesn't scale them

### ISSUE THREAD

**Issue by ntoronto** [ntoronto/pict3d#12]
This is usually only a modeling problem, with the workaround "don't do that" or "adjust the light intensity for the scaled result." But freezing a point light makes its shader think its bounds should be much smaller, which creates undesirable artifacts.

```racket
#lang racket
(require pict3d)
(combine (scale (freeze (light (pos 0 0 -1)
                               (emitted "yellow" 0.25)))
                0.1)
         (rectangle (pos -2 -2 -1)
                    (pos 2 2 -1/2)))
```

One solution is to associate an affine transformation with each light, just like spheres. We'd end up with elliptical light sources as well as spherical, which actually isn't a terrible thing.

**Comment by ntoronto** [comment]
![bad-point-light](https://cloud.githubusercontent.com/assets/489358/6557870/d05a9aee-c64e-11e4-8eb2-45323ecfc304.png)

**Comment by ntoronto** [comment]
Fixed by 56e3cb404a7dd08cf9de4f647e947243ed61270c.
